pages/board/[slug].vue

```diff
@@ -0,0 +1,11 @@
+<script setup lang="ts">
+import { useGetLeaderboardDetail } from 'composables/api/useGetLeaderboardDetail'
+
+const route = useRoute()
+const leaderboardSlug = route.params.slug as string
+const leaderboardDetail = await useGetLeaderboardDetail(leaderboardSlug)
+</script>
+
+<template>
+  <p>Leaderboard Name: {{ leaderboardDetail.data?.name }}</p>
```

Comment:
Possibly not needed, but should we create a fallback if the `data` object doesn't exist?

Reply:
Good catch. Literally forgot about my own documentation:
https://github.com/leaderboardsgg/leaderboard-site/pull/553/files#diff-23b02f15104a6269aea8c1c74c2edefffe95ca0705f5601f57137b6269121329R26

Added a fallback here and on the user page